docs/docs/persistent-volumes.md

```diff
@@ -69,6 +69,58 @@ The default `UpgradeStrategy` for a stateful application is a `minimumHealthCapa
 
 **Note:** For a stateful application, Marathon will never start more instances than specified in the `UpgradeStrategy`, and will kill old instances rather than create new ones during an upgrade or restart.
 
+## Creating a pod with local persistent volumes
+
+### Configuration options
+
+Let's configure a persistent volume with the following options:
+
+```json
+{
+  "name": "pst",
+  "persistent": {
+    "type": "root",
+    "size": 10,
+    "constraints": []
+  }
+}
+```
+
+- `name`: Name of the pod level volume
+- `persistent.type`: The type of mesos disk resource to use; the valid options are `root`, `path`, and `mount`, corresponding to the [valid mesos multi-disk resource types](http://mesos.apache.org/documentation/latest/multiple-disk/).
+- `persistent.size`: The size of the persistent volume in MiBs.
+- `persistent.maxSize`: (not seen above) For `root` mesos disk resources, the optional maximum size of an exclusive mount volume to be considered.
+- `persistent.profileName`: (not seen above) The storage volume profile. Only volumes with the specified profile are used to launch an application. It this option is not given, any volume (with or without a profile) will be used for launching.
+- `persistent.constraints`: Constraints restricting where new persistent volumes should be created. Currently, it is only possible to constrain the path of the disk resource by regular expression.
+
+
+Next, we also need to set the `residency` node in order to tell Marathon to setup a stateful pod. 
+Currently, the only valid option for this is:
+
+```
+"residency": {
+  "taskLostBehavior": "WAIT_FOREVER",
+  "relaunchEscalationTimeoutSeconds": 3600
```

Comment:
According to the code, in order to use persistent volumes, there are two things required:

1. `unreachableStrategy` must be set to `disabled`,
2. `residency` must be defined (surprisingly, none of its properties is actually used throughout the code. Therefore, let's require the following for the time being:

```
"unreachableStrategy": "disabled",
"residency": {
    "taskLostBehaviour": "WAIT_FOREVER"
}
```

Could you please change it accordingly throughout the doc?